apps/studio/src/features/editing-experience/hooks/useTextEditor/constants.ts

```diff
@@ -87,11 +80,24 @@ export const BASE_EXTENSIONS: Extensions = [
       }
     },
   }),
+  History,
+  Italic,
+  Paragraph,
+  Text,
+  Underline,
+]
+
+export const BASE_PROSE_EXTENSIONS: Extensions = [
```

Comment:
nit: uhh now we have `BASE_EXTENSIONS` and `BASE_PROSE_EXTENSIONS` which I think the naming can be quite confusing. Can this be renamed to something else instead? Thinking perhaps just `PROSE_EXTENSIONS` should be sufficient (but can suggest a better name if you have)